Firmware/planner.h

```diff
@@ -71,7 +71,7 @@ union dda_usteps_t
 typedef struct {
   // Fields used by the bresenham algorithm for tracing the line
   // steps_x.y,z, step_event_count, acceleration_rate, direction_bits and active_extruder are set by plan_buffer_line().
-  dda_isteps_t steps_x, steps_y, steps_z, steps_e;  // Step count along each axis
+  dda_isteps_t steps[NUM_AXIS];  // Step count along each axis
```

Comment:
Maybe align the comment here to keep it tidy

Reply:
Done